chromadb/segment/__init__.py

```diff
@@ -70,6 +71,7 @@ def get_metadata(
         limit: Optional[int] = None,
         offset: Optional[int] = None,
         include_metadata: bool = True,
+        request_metadata: Optional[RequestMetadata] = None,
```

Comment:
Is this only optional for single node compat ? I feel like in distributed we always want it no?